images/linux/scripts/installers/homebrew.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  homebrew.sh
+##  Desc:  Installs the Homebrew on Linux
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+
+#  Install the Homebrew on Linux
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+echo 'eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)' >> ~/.profile
```

Comment:
`~/.profile` will be removed after image generation because user will be recreated. Will it work as you expected?

Reply:
moved to /etc/profile.d https://github.com/actions/virtual-environments/pull/589/commits/358c29ef00d560b29d44adeff502bf6c56b52da3